spec/classes/foreman_proxy__tftp_spec.rb

```diff
@@ -15,6 +15,7 @@
 
       it { is_expected.to contain_class('foreman_proxy::tftp::netboot') }
 
+      system_image_root = '/var/lib/foreman-proxy/tftp/system_images'
```

Comment:
This appears unused

Reply:
Do I have to add this different or is it just unnecessary here because I set it elsewhere?